sources/packages/web/src/views/aest/institution/ApproveDenyDesignation.vue

```diff
@@ -16,15 +16,17 @@
       </div>
     </template>
     <template v-slot:footer>
-      <v-btn color="primary" variant="outlined" @click="dialogClosed">
-        Cancel
-      </v-btn>
-      <v-btn
-        @click="submitDesignation()"
-        class="float-right primary-btn-background"
-      >
-        Submit Action
-      </v-btn>
+      <v-row class="m-0 p-0">
```

Comment:
Is this class needed since margin and padding are 0 ?

Reply:
commented above